community/pom.xml

```diff
@@ -5,7 +5,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>org.bonitasoft.web.application</groupId>
 	<artifactId>bonita-admin-application</artifactId>
-	<version>8.0-SNAPSHOT</version>
+	<version>9.0-SNAPSHOT</version>
```

Comment:
?

Reply:
have to rebase this. Was working on the changes

Reply:
Done